packages/starknet-snap/src/index.ts

```diff
@@ -288,7 +288,7 @@ export const onHomePage: OnHomePageHandler = async () => {
       const userAddress = accContract.address;
       const chainId = accContract.chainId;
       const network = getNetworkFromChainId(state, chainId);
-      panelItems.push(row('Address', address(`${userAddress}`)));
+      panelItems.push(row('Address', text(`${userAddress}`)));
```

Comment:
I think you should import copyable and then do two lines: 

```
panelItems.push(text('Address')); 
panelItems.push(copyable(`${userAddress}`)); 
```

Reply:
updated